### ISSUE TITLE
4.2.1 Osmand is really slow at startup

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#14417]
### Description
Since a few days it takes about two minutes until map is shown in detail and can be moved to another place/area. 
Happens on three devices. Data folder is Multiuser 1.

It seems to happen because of the displayed tracks and the favourites.
Without displaying any favourites and tracks Osmand starts up quicker, but still clear visible slower as usual.
I always use lots of favourites and tracks.

### How to reproduce?
See the attached logcat files, one with and the other one without favourites and tracks displayed

### Your Environment
OsmAnd Version: 4.2.1
Android/iOS version: 12
Device model: S20FE, S20+, Tab S6

[logcat with favourites with tracks.log](https://github.com/osmandapp/OsmAnd/files/8796208/logcat.with.favourites.with.tracks.log)
[logcat no favourites no tracks.log](https://github.com/osmandapp/OsmAnd/files/8796214/logcat.no.favourites.no.tracks.log)

**Comment by vshcherb** [comment]
No specific changes done in that area as I know. You could try find any previous version it started failing but as I see it should not be a problem with new build.
P.S. didn't notice anything specific in the logs

**Comment by Rowin63** [comment]
Thanks. 
I would like to go back to a previous 4.1.x version, but (in this case unfortunately) I´m leaving tomorrow for a three weeks motorbike (mainly offroad)-trip, with Osmand prepared as my navigation app (routes, places to see, ...) and I dont want to to any experiments anymore today... 
So I gess I have to deal with this two-minutes-delay at starting of Osmand. I can do anything like searching, open and editing menues etc. from the beginning; but the map itself needs about two minutes until it is displayed. Somethging must have changed recently.

**Comment by Rowin63** [comment]
I have been using osmand for three weeks on tour and tried some different things.
Unfortunately this problem seems to be the result of the large number of tracks stored in osmand [~700 what in parts is the result of importing tracks like TETs as hundreds of single sections).
Deleting all tracks makes osmands start as quick as usual. 
The more tracks, the slower is osmand at startup. This can take up to 2-3 minutes! 
But once this process is done, osmand runs fine.

**Comment by Rowin63** [comment]
problem is no longer relevant.
